packages/block-library/src/comment-author-name/block.json

```diff
@@ -29,7 +29,11 @@
 		},
 		"color": {
 			"gradients": true,
-			"link": true
+			"link": true,
+			"__experimentalDefaultControls": {
```

Comment:
Would it be worth including `link` as one of the defaults here, otherwise if you switch on the "Link to authors URL" setting, it isn't immediately clear where to adjust the link color.